lib/session.js

```diff
@@ -91,6 +92,10 @@ class Session {
     return this.config.waitTimeMultiApproval;
   }
 
+  get ciType() {
+    return this.config.ciType || 'jenkins';
```

Comment:
I think we can make `github-check` the default now?

Reply:
Not yet. If we set `github-check` as the default, ncu will not warn users that Jenkins jobs didn't ran on a `nodejs/node` Pull Request, since Travis and GitHub Actions also publish to GitHub Check.

Reply:
I guess we'll need to add extra logic for Travis v.s. Jenkins status updates, then. Until then it always warns unless you do `ncu-config set ciType github-check` since the bot (or people) no longer posts that comment to the PRs

Reply:
> since the bot (or people) no longer posts that comment to the PRs

Which comment? The bot is still commenting when a CI starts.